lib/room.rb

```diff
@@ -72,16 +73,16 @@ def events
   # Fetch the next 5 events today for this room
   def fetch_events(calendar_id)
     response = service.list_events(calendar_id,
```

Comment:
I'm surprised that this is the auto-fixed format, I think rubocop would yell different things. OTOH rubocop is not ideal either, it's just that this seems an in-between state where the arguments are not aligned to each other (the positional argument is not aligned with the keyword arguments), but eh.